app/services/base_workflow_service.rb

```diff
@@ -5,31 +5,31 @@ def initialize(assessment)
     @calculation_period = CalculationPeriod.new(@submission_date)
   end
 
-  def applicant
-    @applicant ||= @assessment.applicant
-  end
+  private
 
-  def bank_accounts
-    @bank_accounts ||= @assessment.bank_accounts
-  end
+  attr_reader :assessment
 
-  def non_liquid_assets
-    @non_liquid_assets ||= @assessment.non_liquid_assets
+  def applicant
+    @applicant ||= @assessment.applicant
   end
```

Comment:
Both `applicant` and `capital_summary` can be delegated from `assessment`. Then `main_home`, `additional_properties`, `liquid_capital_items` and `vehicles` can be delegated from `capital_summary`.